### PR TITLE
IMAP Migration in the admin center update

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/2931

image

@JoanneHendrickson Do you agree with this change? Thank you

### CODE REVIEW

**Comment by scanum** [comment]
[@JohanFreelancer9](https://github.com/JohanFreelancer9) Copy editing is needed for this PR. Thanks.

**Comment by scanum** [comment]
[@JohanFreelancer9](https://github.com/JohanFreelancer9) Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/mailbox-migration/migrating-imap-mailboxes/imap-migration-in-the-admin-center.md [hunk 1]
```diff
@@ -30,89 +30,17 @@ manager: serdars
...
-
 > [!IMPORTANT]
 > Before you can use an IMAP migration for your users, they must have been first added to your Microsoft 365 or Office 365 organization. For instructions, see [Add users individually or in bulk](/microsoft-365/admin/add-users/add-users).
 
 Before you migrate, read [What you need to know about migrating your IMAP mailboxes to Microsoft 365 or Office 365](migrating-imap-mailboxes.md).
 
-To perform an IMAP migration by using the Exchange admin center (EAC), see [Migrate other types of IMAP mailboxes to Microsoft 365 or Office 365](migrate-other-types-of-imap-mailboxes.md).
+> [!IMPORTANT]
+> IMAP migration in the Microsoft 365 admin center has been replaced by IMAP migration by using the Exchange admin center (EAC).
+> To perform an IMAP migration by using the Exchange admin center (EAC), see [Migrate other types of IMAP mailboxes to Microsoft 365 or Office 365](migrate-other-types-of-imap-mailboxes.md).
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mailbox-migration/migrating-imap-mailboxes/imap-migration-in-the-admin-center.md:40]:
```suggestion
> IMAP migration in the Microsoft 365 admin center has been replaced by IMAP migration by using the Exchange admin center (EAC). To perform an IMAP migration by using the EAC, see [Migrate other types of IMAP mailboxes to Microsoft 365 or Office 365](migrate-other-types-of-imap-mailboxes.md).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by get-itips** [comment]
@chrisda What do you think? Thank you

**Comment by scanum** [comment]
@chrisda PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**